core/src/main/java/hudson/model/ViewDescriptor.java

```diff
@@ -79,6 +79,7 @@ protected ViewDescriptor() {
      */
     @Restricted(DoNotUse.class)
     public AutoCompletionCandidates doAutoCompleteCopyNewItemFrom(@QueryParameter final String value, @AncestorInPath ItemGroup<?> container) {
+        // TODO do we need a permissions check here?
```

Comment:
It does not create anything. `AutoCompletionCandidates#ofJobNames` suggests only what can be read so do not think there is an information disclosure here, either. Imho no.